frontend/build-tc.sh

```diff
@@ -1,2 +1,5 @@
 #!/bin/bash
 echo ${BUILD_NUMBER:-DEV} > conf/build.txt
+# put the script needed on the machine into the package so it ends up on the instance via riffraff
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo "##teamcity[publishArtifacts '$DIR/instance => .']"
```

Comment:
this magic println pushes the instance dir into the build artifacts dir, which riffraff eventually pushes onto the box so we can access it at startup